packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -194,6 +194,17 @@ const cookieTest = async (req, res) => {
     res.json(jsonFromRequest(req))
 }
 
+/**
+ * Express handler that sets a response header and returns a JSON response with
+ * diagnostic values.  Use ?name=test-name&value=test-value to set a response header.
+ */
+const responseHeader = async (req, res) => {
+    if (Object.hasOwn(req.query, 'name') && Object.hasOwn(req.query, 'value')) {
+        res.set(req.query.name, req.query.value)
+    }
+    res.json(jsonFromRequest(req))
```

Comment:
Did you mean to return the response object and not the request object?

Reply:
No this app always echos back the request to the user so I am continuing to have all endpoints follow the same pattern. For the e2e tests I plan to check the response headers directly (not from the response body).